src/Agent.Sdk/ProcessInvoker.cs

```diff
@@ -189,7 +189,8 @@ public ProcessInvoker(ITraceWriter trace)
                 redirectStandardIn: redirectStandardIn,
                 inheritConsoleHandler: inheritConsoleHandler,
                 keepStandardInOpen: false,
-                cancellationToken: cancellationToken);
+                cancellationToken: cancellationToken,
+                decreaseProcessPriority: false);
```

Comment:
i thought what we talked about is something like `hightPriorityProcess: false`, then the only caller you need to change is when agent launch worker, set it to `true`. all other processes invoke in the agent will be low priority.

Reply:
I updated it to use that pattern. Seemed to simplify things :)